client/src/ClientGame.cpp

```diff
@@ -19,14 +19,18 @@ ClientGame::ClientGame()
 void ClientGame::updateAnimations(std::bitset<NUM_STATES> movementEntityStates[]) {
     for (unsigned int i = 0; i < NUM_PLAYER_ENTITIES; i++) {
         unsigned int movementIndex = playerIndices[i];
-        if (movementEntityStates[movementIndex][MOVING_HORIZONTALLY]) {
+        if (movementEntityStates[i][MOVING_HORIZONTALLY]) {
```

Comment:
Why did you switch to using i here? Did something change about how movementEntityStates works? The point of movementIndex = playerIndices[i] was to go from an index into the players (0, 1, 2, or 3) to an index into the general list of moving entities. I know we do put the players at 0, 1, 2, and 3 in the movement list as well, but I felt like we were less likely to introduce bugs this way and would extend better to doing similar things to other categories of entities.

Reply:
ur right!

once upon time there was a bug that you can't see other player's shooting animation. i thought this was the cause (but its not). 

I'll be using playerIndices from now on, it sets a good ground to extend other types of moving entities